app/models/user.rb

```diff
@@ -129,6 +129,10 @@ def self.name_format
 
   attr_exportable :firstname, :lastname, :mail, :description, :fullname, :name => ->(user) { user.login }, :ssh_authorized_keys => ->(user) { user.ssh_keys.map(&:to_export_hash) }
 
+  def as_json(options = {})
+    super.as_json(options).tap { |u| u['user']['name'] = name }
```

Comment:
doesn't `super` already call `as_json(options)` on the parent class?

Reply:
yup, updated, calling just `super`